elements/pf-date-picker/pf-calendar.ts

```diff
@@ -0,0 +1,269 @@
+import { LitElement, html } from 'lit';
+import { customElement } from 'lit/decorators/custom-element.js';
+import { property } from 'lit/decorators/property.js';
+import { ref, createRef } from 'lit/directives/ref.js';
+import styles from './pf-calendar.css';
+import { getFormattedDate, getMonthNamesFromLocale } from './date-picker-helper.js';
+import { ComposedEvent } from '@patternfly/pfe-core/core.js';
+
+export interface FocusedDateValues{
+  day: number;
+  month: number;
+  year: number;
+}
+
+export interface CalendarDateValues{
+  day: number;
+  month: number;
+  year: number;
+  isCurrentMonth: boolean;
+}
+
+/**
+ * Date Calendar
+ * @slot - Place element content here
+ */
+
+export class DayChangeEvent extends ComposedEvent {
+  constructor(public event: Event, public day: number, public month: number, public year: number) {
+    super('daySelected');
+  }
+}
+
+export class FocusChangeEvent extends ComposedEvent {
+  constructor(public dateToBeFocusedRef: HTMLButtonElement | undefined) {
+    super('setDateFocus');
+  }
+}
+
+export class KeyboardNavigationFocusEvent extends ComposedEvent {
+  constructor(public event: Event, public day: number, public month: number, public year: number) {
+    super('onCalendarKeydown');
+  }
+}
+
+/**
+ * Calendar
+ * @slot - Place element content here
+ */
+@customElement('pf-calendar')
+export class PfCalendar extends LitElement {
+  static readonly styles = [styles];
+
+  private currentDate: Date = new Date();
```

Comment:
prefer ecmascript private fields where possible
```suggestion
  #currentDate = new Date();
```

also note that in this case you can do without the type annotation, TS will figure it out from the initializer on the RHS

Reply:
Added ES6 private fields